Source/AppScaffolding/LibationScaffolding.cs

```diff
@@ -405,25 +405,58 @@ public static void migrate_to_v6_6_9(Configuration config)
 
 		public static void migrate_from_7_10_1(Configuration config)
 		{
```

Comment:
I figure this stays in for a few release cycles. Gets the regular and irregular users databases migrated. Then it can be deleted. After that, the app still won't crash with a bad DB, but if they want all their stuff shown they have to delete and resync.